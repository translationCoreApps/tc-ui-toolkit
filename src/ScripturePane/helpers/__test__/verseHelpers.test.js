import fs from 'fs';
import path from 'path';
import * as verseHelpers from '../verseHelpers';

describe('verseHelpers.verseArray', () => {
  it('should succeed with heb-12-27.el-x-koine', () => {
    generateTest('heb-12-27.el-x-koine', 'ugnt');
  });
  it('should succeed with heb-12-27.el-x-koine and tisch should not be clickable', () => {
    generateTest('heb-12-27.el-x-koine', 'tisch');
  });
  it('should succeed with mat-4-6', () => {
    generateTest('mat-4-6', 'ult');
  });
  it('should succeed with luke-22-30.ult', () => {
    const contextId = {
      'reference': {
        'bookId': 'luk',
        'chapter': 22,
        'verse': 30,
      },
      'tool': 'translationWords',
      'groupId': '12tribesofisrael',
      'quote': [
        {
          'word': 'δώδεκα',
          'occurrence': 1,
        },
        {
          'word': 'φυλὰς',
          'occurrence': 1,
        },
        {
          'word': 'κρίνοντες',
          'occurrence': 1,
        },
        {
          'word': 'τοῦ',
          'occurrence': 1,
        },
        {
          'word': 'Ἰσραήλ',
          'occurrence': 1,
        },
      ],
      'strong': [
        'G14270',
        'G54430',
        'G29190',
        'G35880',
        'G24740',
      ],
      'occurrence': 1,
    };
    generateTest('luke-22-30.ult', 'ult', contextId);
  });
  it('should succeed with jhn-6-21-en-t4t', () => {
    generateTest('jhn-6-21-en-t4t', 't4t');
  });
});

//
// helpers
//

/**
 * Reads a usfm file from the resources dir
 * @param {string} filePath relative path to usfm file
 * @return {string} sdv
 */
export const readUSFM = filePath => {
  const fullPath = path.join('./src/ScripturePane/helpers/__test__/fixtures', filePath);
  console.log(path.resolve(fullPath));
  return fs.readFileSync(fullPath, 'UTF-8').toString();
};

/**
 * Reads a json file from the resources dir
 * @param {string} filePath relative path to json file
 * @return {object} json object
 */
export const readJSON = filePath => JSON.parse(readUSFM(filePath));

/**
 * Generator for testing json to usfm migration
 * @param {string} name - the name of the test files to use. e.g. `valid` will test `valid.usfm` to `valid.json`
 * @param {string} bibleId
 * @param {Object} contextId
 */
const generateTest = (name, bibleId, contextId='') => {
  const input = readJSON(`${name}.json`);
  contextId = contextId || name;
  expect(input).toBeTruthy();
  // const expectedBaseName = expectedName ? expectedName : name;
  // const expected = readUSFM(`${expectedBaseName}.usfm`);
  // expect(expected).toBeTruthy();
  const output = verseHelpers.verseArray(input, bibleId, contextId);
  expect(output).toMatchSnapshot();
};
