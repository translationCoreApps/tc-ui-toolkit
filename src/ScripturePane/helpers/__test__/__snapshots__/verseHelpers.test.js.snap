// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`verseHelpers.verseArray should succeed with heb-12-27.el-x-koine 1`] = `
Array [
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τὸ
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      δὲ
    </span>
  </span>,
  <span>
    ,
“
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἔτι
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἅπαξ
    </span>
  </span>,
  <span>
    ”,
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      δηλοῖ
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τῶν
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      σαλευομένων
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μετάθεσιν
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ὡς
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      πεποιημένων
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἵνα
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μείνῃ
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τὰ
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μὴ
    </span>
  </span>,
  <span
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      σαλευόμενα
    </span>
  </span>,
  <span>
    . 
  </span>,
]
`;

exports[`verseHelpers.verseArray should succeed with heb-12-27.el-x-koine and tisch should not be clickable 1`] = `
Array [
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τὸ
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      δὲ
    </span>
  </span>,
  <span>
    ,
“
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἔτι
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἅπαξ
    </span>
  </span>,
  <span>
    ”,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      δηλοῖ
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τῶν
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      σαλευομένων
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μετάθεσιν
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ὡς
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      πεποιημένων
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      ἵνα
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μείνῃ
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      τὰ
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      μὴ
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      σαλευόμενα
    </span>
  </span>,
  <span>
    . 
  </span>,
]
`;

exports[`verseHelpers.verseArray should succeed with jhn-6-21-en-t4t 1`] = `
Array [
  <span>
    We were glad to take him into the boat. As soon as 
  </span>,
  <span>
    we
  </span>,
  <span>
     did that, the boat reached the shore where 
  </span>,
  <span>
    we
  </span>,
  <span>
     were going! 
  </span>,
]
`;

exports[`verseHelpers.verseArray should succeed with luke-22-30.ult 1`] = `
Array [
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      that
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      may
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      eat
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      and
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      drink
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      at
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      my
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      table
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      in
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      my
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      kingdom
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      and
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      will
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      sit
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      on
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      thrones
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
      judging
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      the
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
      twelve
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
      tribes
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
      of
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "var(--highlight-color)",
        }
      }
    >
      Israel
    </span>
  </span>,
  <span>
    .
  </span>,
  <span
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
     
  </span>,
  <span
    style={
      Object {
        "backgroundColor": "var(--highlight-color)",
      }
    }
  >
    
  </span>,
]
`;

exports[`verseHelpers.verseArray should succeed with mat-4-6 1`] = `
Array [
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      and
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      said
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      to
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      him
    </span>
  </span>,
  <span>
    ,"
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      If
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      are
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      the
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      Son
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      of
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      God
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      throw
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      yourself
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      down
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      for
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      it
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      is
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      written
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
     
  </span>,
  <span>
    '
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      He
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      will
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      command
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      his
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      angels
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      to
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      take
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      care
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      of
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    ,'
  </span>,
  <span
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
     
  </span>,
  <span>
    and, 
  </span>,
  <span
    style={
      Object {
        "backgroundColor": "transparent",
      }
    }
  >
     
  </span>,
  <span>
    '
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
      
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      They
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      will
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      lift
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      up
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      in
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      their
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      hands
    </span>
  </span>,
  <span>
    ,
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      so
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      that
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      you
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      will
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      not
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      hit
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      your
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      foot
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      against
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      a
    </span>
  </span>,
  <span>
    <span
      style={
        Object {
          "backgroundColor": "transparent",
        }
      }
    >
       
    </span>
    <span
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      stone
    </span>
  </span>,
  <span>
    .'"
  </span>,
  <span>
    
  </span>,
]
`;
